packages/tokens/src/bin/tsconfig.json

```diff
@@ -3,6 +3,10 @@
   "include": [
     "**/*",
   ],
+  "exclude": [
```

Comment:
I have this warning appearing inside tests in `src/bin/tests` ( they do not have any tsconfig file referencing them ) 

<img width="578" alt="Capture d’écran 2023-07-03 à 10 40 15" src="https://github.com/openfun/cunningham/assets/9628870/2d443a02-0351-4ea3-9973-14bea84f3189">

Hence, the tests files inside `src/bin/Generators` are still getting compiled into `dist`.

That being said, few points:
- We could do this job in another PR ( it seems to be a different subject ) 
- We could simply rm -rf `*.spec.js` files from `dist` after build.
- There's maybe an option for tsconfig file to not generate js files for specific patterns ( but still reference them in order to make them interpretable by IDE + jest ) 

WDYT ?